src/targets/gpu/include/migraphx/gpu/miopen.hpp

```diff
@@ -75,21 +75,31 @@ inline tensor_descriptor make_tensor(const migraphx::shape& os, bool pack = fals
     return t;
 }
 
+inline void insert_attribute_1d_dims(std::vector<int>& padding,
+                                     std::vector<int>& stride,
+                                     std::vector<int>& dilation)
+{
+    padding.insert(padding.begin(), 0);
+    stride.insert(stride.begin(), 1);
+    dilation.insert(dilation.begin(), 1);
+}
+
 template <class T>
 inline convolution_descriptor make_conv(const T& op)
 {
     auto c = make_obj<convolution_descriptor>(&miopenCreateConvolutionDescriptor);
     miopenConvolutionMode_t c_mode = miopenConvolution;
     if(op.group > 1)
         c_mode = miopenGroupConv;
-    miopenInitConvolutionDescriptor(c.get(),
-                                    c_mode,
-                                    op.padding[0],
-                                    op.padding[1],
-                                    op.stride[0],
-                                    op.stride[1],
-                                    op.dilation[0],
-                                    op.dilation[1]);
+    std::vector<int> padding(op.padding.begin(), op.padding.end());
```

Comment:
Padding, stride, and dilation is represented as `std::size_t` not `int`.

Reply:
The miopen call below requires the attribute data to be `int*`, so I just copied the values as ints here.

Reply:
Ok I see.